docker/build-project.sh

```diff
@@ -133,14 +137,15 @@ fi
 # Push buildhistory into machine-specific branch in the master buildhistory
 #
 if [ ! -z ${JOB_NAME+x} ]; then
```

Comment:
1. this block runs after bitbake, so what you really mean is to put BH enablement into
block above, near "# Prepare for buildhistory generation" ?

2. should we do these BH enablement changes also in current PR ?